webpack/webpack.config.cjs

```diff
@@ -155,7 +154,7 @@ module.exports = function(env, argv) {
     optimization: {
       minimize: true,
       minimizer: [
-        new TerserWebpackPlugin({
+        () => ({
```

Comment:
Do you think it's worth a comment here about why this is not what the previous code was?